README.md

```diff
@@ -5,6 +5,9 @@ Collection of [syzkaller](https://github.com/google/syzkaller) bug reproducers f
 Obtained from [syzbot](https://github.com/google/syzkaller/blob/master/docs/syzbot.md)
 using [syz-reprolist](https://github.com/google/syzkaller/blob/64612bfd3c83d7246467cd9731f01a36b6331598/tools/syz-reprolist/reprolist.go).
 
+Collect reproducers from subdirs.
```

Comment:
I think it would be useful to mention that this is supposed to work with syz-manager workdir. Perhaps something like:
```
To collect C reproducers from syz-manager workdir, run [./collect_reprogs.py <path/**/*.cprog>](collect_reprogs.py).
```

Reply:
done.